pandas/tests/tools/test_to_datetime.py

```diff
@@ -71,6 +71,16 @@ def test_to_datetime_format(self, cache):
                 else:
                     tm.assert_index_equal(result, expected)
 
+    def test_to_datetime_format_some_invalid_numbers(self):
+        s = Series([20181223, 20161017, -9900, 20171220, 20170511])
+        expected = Series(['2018-12-23', '2016-10-17', -9900,
```

Comment:
I'd construct `expected` in a single step here - this is a little hard to parse IMO